cozepy/websockets/chat/__init__.py

```
@@ -130,10 +132,49 @@
+        role: str
+        # 消息内容的类型，支持设置为：text：文本。object_string：多模态内容，即文本和文件的组合、文本和图片的组合。
+        content_type: str
+        # 消息的内容，支持纯文本、多模态（文本、图片、文件混合输入）、卡片等多种类型的内容。当 content_type 为 object_string时，content 的结构和详细参数说明请参见object_string object。
+        content: str
+
+    event_type: WebsocketsEventType = WebsocketsEventType.CONVERSATION_MESSAGE_CREATE
+    data: Data
+
+
+# req
+class ConversationClear(WebsocketsEvent):
```

Pylint (3.3.7)

[refactor] Too few public methods (1/2)

(R0903)